src/main/java/step4/model/Lottos.java

```diff
@@ -0,0 +1,64 @@
+package step4.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Lottos {
+    private final List<Lotto> lottos;
+
+    public Lottos() {
+        this.lottos = new ArrayList<>();
+    }
+
+    public Lottos(Lotto lotto) {
+        this.lottos = Stream.of(lotto).collect(Collectors.toList());
+    }
```

Comment:
한 객체로 리스트를 생성하는 것이 목적이라면 싱글톤 리스트를 이용해보시면 좋을 것같아요!!
```suggestion
    public Lottos(Lotto lotto) {
        this.lottos = Collections.singletonList(lotto);
    }
```